Asher-Paris/Asher-Paris.md

```diff
@@ -7,11 +7,11 @@ quote: "We should reward people, not ridicule them, for thinking the impossible.
 quoteAuthor: "Winston Churchill"
 ---
 
-Asher is a SSW copywriter and Marketing Assistant who loves anything and everything copywriting. Why? Because the opportunityisnowhere.
+Asher is a SSW copywriter and Marketing Assistant who loves anything and everything to do with copywriting. Why? Because the opportunityisnowhere.
```

Comment:
```suggestion
Asher is an SSW copywriter and Marketing Assistant who loves anything and everything to do with copywriting. Why? Because the opportunityisnowhere.
```